internal/utils/config.go

```diff
@@ -168,30 +166,16 @@ func MakeConfig() (*types.Config, error) {
 		BlackWhitelistNamespace:         getEnv("BLACK_WHITELIST_NAMESPACE", "default"),
 	}
 
-	// TODO: Remove validation through ozzo-validation
-	err = validation.ValidateStruct(config,
-		validation.Field(&config.KubeToken, validation.Required),
-		validation.Field(&config.KubeCa, validation.Required, is.Base64),
-		validation.Field(&config.PublicApiServerURL, validation.Required, is.URL),
-	)
-	// TODO: Get rid of Check method
-	Check(err)
-
-	errLdap := validation.ValidateStruct(&ldapConfig,
-		validation.Field(&ldapConfig.UserBase, validation.Required, validation.Length(2, 200)),
-		validation.Field(&ldapConfig.GroupBase, validation.Required, validation.Length(2, 200)),
-		validation.Field(&ldapConfig.Host, validation.Required, is.URL),
-		validation.Field(&ldapConfig.BindDN, validation.Required, validation.Length(2, 200)),
-		validation.Field(&ldapConfig.BindPassword, validation.Required, validation.Length(2, 200)),
-	)
+	err = validateConfig(config)
```

Comment:
validate config inside MakeConfig is very confusing.

Can't we simply use the MakeConfig as a constructor based on the config provided in the main, and validate all we need directly?

I am fine with splitting the function to make it clearer, so I am fine with validateLdapConfig. However, I believe that validation can be done immediately in a single step.

Also, we don't need to create validation functions, unless they are used multiple times and make it easier to read. 

In all cases, don't forget to add tests...